deselected_tests.yaml

```diff
@@ -25,6 +25,20 @@
 #  will exclude deselection in versions 0.18.1, and 0.18.2 only.
 
 deselected_tests:
+  # sklearn 1.6 unsupported features
+  - linear_model/tests/test_base.py::test_linear_regression_sample_weight_consistency[42-True-None-X_shape1]
```

Comment:
Some of these probably need tickets for follow-on analysis. Can be assigned to @Vika-F , @david-cortes-intel or @DDJHB since they have the most linreg/ridge experience.

Reply:
I created investigation ticket.